webapp/tests/utils.py

```diff
@@ -15,3 +20,23 @@ def create_session_form_data(data):
 def create_and_activate_user(idnr, dob, request_id, unlock_code):
     create_user(idnr, dob, request_id)
     find_user(idnr).activate(unlock_code)
+
+
+@pytest.fixture
+def production_flask_env():
+    current_configuration_value = os.environ.get('FLASK_ENV')
+    os.environ['FLASK_ENV'] = 'production'
+
+    yield current_app
+
+    os.environ['FLASK_ENV'] = current_configuration_value
+
+
+@pytest.fixture
+def staging_flask_env():
+    current_configuration_value = os.environ.get('FLASK_ENV')
+    os.environ['FLASK_ENV'] = 'staging'
+
+    yield current_app
+
+    os.environ['FLASK_ENV'] = current_configuration_value
```

Comment:
I would keep fixtures that are used only by tests in a single module inside that module file _until_ you find a case to use them elsewhere. Otherwise our collection of _shared_ fixtures becomes hard to comprehend very quickly. Or do you see a need to use these elsewhere very soon?

Reply:
It was shared because we needed it also for the decorator tests. Because that is now removed, I moved these fixtures into the module where they are used